com.leinardi.gst.json

```diff
@@ -0,0 +1,158 @@
+{
+  "app-id": "com.leinardi.gst",
+  "command": "gst",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.34",
+  "sdk": "org.gnome.Sdk",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--device=dri",
+    "--share=network",
+    "--talk-name=org.freedesktop.Flatpak",
+    "--filesystem=/sys:ro",
+    "--env=PATH=/usr/bin:/bin:/usr/sbin:/sbin:/app/bin:/app/sbin",
+    "--env=LD_LIBRARY_PATH=/app/lib/x86_64-linux-gnu/:/app/lib/i386-linux-gnu/:/app/lib64:/app/lib"
```

Comment:
Is this actually needed?

Reply:
Just tested, it seems not needed. Removed